documentation/source/newsfragments/1683.feature.rst

```diff
@@ -0,0 +1 @@
+Add Gitpod support. Can by tried under https://gitpod.io/#https://github.com/cocotb/cocotb
```

Comment:
Unsure whether we should include this in our release notes - it's not really something we "release", especially since it points to our unreleased master.

Reply:
Especially considering https://github.com/cocotb/cocotb/pull/1683#issuecomment-617253914